README.md

```diff
@@ -85,6 +85,10 @@ For example, [here](https://github.com/pointfreeco/isowords/blob/main/App/Previe
 This repo contains both the client and server code for running the entire [isowords](https://www.isowords.xyz) application, as well as an extensive test suite. To get things running:
 
 1. Make sure [`git-lfs`](https://git-lfs.github.com) is installed so that app assets (images, etc.) can be fetched.
+    1. Run 'git lfs install' in isowords directory to complete installation or you may encounter the following error:
+        ```sh
+        Skipping object checkout, Git LFS is not installed
+        ```
```

Comment:
What do you think of spelling out the instructions more specifically?

```suggestion
1. Make sure [`git-lfs`](https://git-lfs.github.com) is installed so that app assets (images, etc.) can be fetched. For example, with [Homebrew](https://brew.sh):

    ```sh
    $ brew install git-lfs  # Download and install Git LFS
    $ git lfs install       # Set up Git LFS for your user
    ```
```